docs/integrations/app-integrations/symbl.mdx

```diff
@@ -0,0 +1,84 @@
+---
```

Comment:
Please add this page to `mint.json` and `/integrations/data-overview#applications`.